cs3api4lab/locks/cs3.py

```diff
@@ -0,0 +1,95 @@
+import time
+
+from cs3api4lab.locks.base import LockBase
+from cs3api4lab.utils.file_utils import FileUtils
+from cs3api4lab.api.cs3_user_api import Cs3UserApi
+from cs3api4lab.exception.exceptions import FileLockedError
+
+import cs3.storage.provider.v1beta1.provider_api_pb2 as storage_api
+import cs3.types.v1beta1.types_pb2 as cs3_types
+import cs3.identity.user.v1beta1.resources_pb2 as id_res
+import cs3.storage.provider.v1beta1.resources_pb2 as storage_resources
+import cs3.rpc.v1beta1.code_pb2 as cs3code
+
+import google.protobuf.json_format as json_format
+
+
+class Cs3(LockBase):
+
+    def __init__(self, log, config):
+        super().__init__(log, config)
+        self.user_api = Cs3UserApi(log)
+        self.config = config
+
+    def handle_locks(self, file_path, endpoint):
+        ref = FileUtils.get_reference(file_path, endpoint)
+        lock = self._get_lock(ref)
```

Comment:
I see that the naming of the internal methods are not consistent (i.e `_set_lock` vs `_lock_file`).
They don't necessarily need to be, but I would prefer if they were. It makes things easier to understand. Please check as well that we can make all methods aligned.
(Ofc this is not urgent, but I don't expect it to be a big change...)